pkgs/development/libraries/librealsense/default.nix

```
@@ -1,4 +1,9 @@
-{ stdenv, fetchFromGitHub, cmake, libusb1, ninja, pkgconfig }:
+{ stdenv, config, lib, fetchFromGitHub, cmake, libusb1, ninja, pkgconfig
+, cudaSupport ? config.cudaSupport or false, cudatoolkit
+, enablePython ? false, pythonPackages }:
```

looks like you don't care so much for the packageset, as much as python
```suggestion
, enablePython ? false, python }:
```

---

in fact it should compile a python module, so it should basically call buildPythonPackage, except that the job is done by their cmake process instead of a setup.py file